src/components/resumes/Details.jsx

```diff
@@ -65,7 +107,7 @@ function Details() {
     setOpenModal(true);
   };
 
-  useEffect(() => {
+  fetch(() => {
```

Comment:
I not sure about this change, can you explain me please?

Reply:
it's a error I have changed to useEffect